Makefile

```
@@ -116,7 +117,15 @@ installer:
 		--frontend-opt target=$@ \
 		$(COMMON_ARGS)
 	@docker load < build/$@.tar
-	@docker run --rm -v /dev:/dev -v $(PWD)/build:/out --privileged autonomy/$@:$(TAG) image -l
+
+image-gcloud: installer
+	@docker run --rm -v /dev:/dev -v $(PWD)/build/gcloud:/out --privileged $(DOCKER_ARGS) autonomy/installer:$(TAG) image -l \
+	-f -p googlecloud -u none -e 'random.trust_cpu=on'
+	@mv $(PWD)/build/gcloud/image.raw $(PWD)/build/gcloud/disk.raw
+	@tar -C $(PWD)/build/gcloud -Sczf talos.tar.gz disk.raw
```

I made a typo in my previous build. Last night I needed this:
```suggestion
	@tar -C $(PWD)/build/gcloud -Sczf $(PWD)/build/gcloud/talos.tar.gz disk.raw
```